runelite-client/src/main/java/net/runelite/client/plugins/xptracker/XpInfoBox.java

```diff
@@ -243,6 +251,36 @@ void update(boolean updated, boolean paused, XpSnapshotSingle xpSnapshotSingle)
 		SwingUtilities.invokeLater(() -> rebuildAsync(updated, paused, xpSnapshotSingle));
 	}
 
+	void toggleHeaderPanel(JPanel progressWrapper)
+	{
+		int progressSkillIconPosition = 0;
```

Comment:
This can be made into a class constant instead of a method field as it is never changed.